src/lib/openapi/spec/event-search-query-parameters.ts

```diff
@@ -0,0 +1,105 @@
+import type { FromQueryParams } from '../util/from-query-params';
+
+export const eventSearchQueryParameters = [
+    {
+        name: 'query',
+        schema: {
+            type: 'string',
+            example: 'admin@example.com',
+        },
+        description:
+            'Find events by a free-text search query. The query will be matched against the event type and the event data payload (if any).',
+        in: 'query',
+    },
+    {
+        name: 'feature',
+        schema: {
+            type: 'string',
+            example: 'IS:myfeature',
+            pattern: '^(IS|IS_ANY_OF):(.*?)(,([a-zA-Z0-9_]+))*$',
+        },
+        description:
+            'Filter by feature name using supported operators: IS, IS_ANY_OF',
+        in: 'query',
+    },
+    {
+        name: 'project',
+        schema: {
+            type: 'string',
+            example: 'IS:default',
+            pattern: '^(IS|IS_ANY_OF):(.*?)(,([a-zA-Z0-9_]+))*$',
+        },
+        description:
+            'Filter by projects ID using supported operators: IS, IS_ANY_OF.',
+        in: 'query',
+    },
+    {
+        name: 'type',
+        schema: {
+            type: 'string',
+            example: 'IS:change-added',
+            pattern: '^(IS|IS_ANY_OF):(.*?)(,([a-zA-Z0-9_]+))*$',
+        },
+        description:
+            'Filter by event type using supported operators: IS, IS_ANY_OF.',
+        in: 'query',
+    },
+
+    {
+        name: 'createdBy',
+        schema: {
+            type: 'string',
+            example: 'IS:2',
+            pattern: '^(IS|IS_ANY_OF):(.*?)(,([a-zA-Z0-9_]+))*$',
+        },
+        description:
+            'The ID of event creator to filter by. The creators can be specified with an operator. The supported operators are IS, IS_ANY_OF.',
```

Comment:
Can we bring this in line with the other is|is_any_of operators?

```suggestion
            'Filter by the ID of the event creator, using supported operators: IS, IS_ANY_OF.',
```